app/views/coronavirus_landing_page/components/hub/_page_header.html.erb

```diff
@@ -30,22 +30,44 @@
       </div>
     </div>
   </div>
-
   <div class="covid__page-header-light covid__page-header-light--hub">
     <div class="govuk-width-container">
       <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
-          <%= render "govuk_publishing_components/components/heading", {
-            text: details.header_section["pretext"],
-            font_size: 19,
-            margin_bottom: 3,
-            inverse: true
-          } %>
-          <ul class="covid__list covid__list--header">
-            <% details.header_section["list"].each do | item | %>
-              <li class="covid__list-item"><%= item %></li>
-            <% end %>
-          </ul>
+          <% if params[:hub_slug] == "education-and-childcare" %>
```

Comment:
I think this new content should be put into it's own partial - that is separated out from the main content. So this part of this page becomes:
```
<% if params[:hub_slug] == "education-and-childcare" %>
  <%= render "govuk_publishing_components/components/hub/education_and_childcare" %>
<% else %>
```

Reply:
Thanks Rob. I'm planning to revert this PR once the content item is updated, and update the `hub/_page_header` partial to pick up the new keys if they're present. So it makes sense to leave as is for the very short time before I undo it! The rendering of the hub pages all happens magically without needing to check what the slug is or if it changes. I'd rather not have a special partial just for the education hub as it will be largely identical to the other hub pages save for reading from a couple of new keys.

Reply:
Personally, the fact that I wanted to remove the content later would make me more inclined to put it into a separate partial, as that would make it easier to identify what needed to be removed. But that may just be me, so fine to carry on as you are.